test/cutensor.jl

```diff
@@ -184,9 +184,11 @@ end
             C  = collect(dC)
             @test C == permutedims(A, p) # exact equality
             Csimple = similar(A, eltyC, dimsC...)
-            CUDA.Mem.pin(Csimple)
-            Csimple = CUTENSOR.permutation!(one(eltyA), A, indsA, Csimple, indsC)
-            @test Csimple == permutedims(A, p) # exact equality
+            if eltyA != Float16
```

Comment:
since the failure only happened on CUDA 10.1, is it not better to version-gate this too?
of course, seeing how it still happens it's probably another test that fails.